tests/commands/new/test_validate_app_name.py

```diff
@@ -9,7 +9,6 @@
         'hello42world',
         '42helloworld',  # ?? Are we sure this is correct?
         'hello_world',
-        'hello-world',
```

Comment:
Why has this test case been removed? It seems like an interesting alternative that should be included, both here, and in the test of `is_valid_app_name`.